src/Backend/Modules/MediaLibrary/Js/MediaLibraryHelper.js

```diff
@@ -841,15 +1032,9 @@ jsBackend.mediaLibraryHelper.upload =
             validation: {
                 allowedExtensions: jsBackend.data.get('MediaLibrary.mediaAllowedExtensions')
             },
-            scaling: {
-                includeExif: true,
-                sendOriginal: false,
-                sizes: [
-                    {name: "", maxSize: 3000}
```

Comment:
We need a limit to the image size, otherwise "thumbnail generating" will reach maximum memory limit and things will break.
=> In your code you have written `{name: "", maxSize: 1} // used to trigger the cropper`

I had set it to 3000 (= pixels) after some testing rounds.